src/page-layout/NoticesLayout/components/Filter/AddressSelector/AddressSelector.tsx

```diff
@@ -0,0 +1,50 @@
+import classNames from "classnames/bind";
+
+import { ADDRESSES } from "@/common/constants";
+import remove from "../../../utils/remove";
+
+import styles from "./AddressSelector.module.scss";
+
+const cn = classNames.bind(styles);
+
+interface AddressSelectorProps {
+  value: string[];
+  setAddress: (address: string[]) => void;
+}
+
+export default function AddressSelector({ value, setAddress }: AddressSelectorProps) {
+  const handleAdd = (badge: string) => {
+    if (!value.includes(badge)) {
+      setAddress([...value, badge]);
+    }
+  };
+
+  const handleDelete = (badge: string) => {
+    setAddress(remove<string>(value, badge));
+  };
+
+  return (
+    <div className={cn("container")}>
+      <div className={cn("selectBoxWrapper")}>
+        <ul className={cn("selectBox")}>
+          {ADDRESSES.map((address: string) => (
```

Comment:
타입이 자동으로 추론되기 때문에 사용하지 않아도 좋습니다~
```suggestion
          {ADDRESSES.map((address) => (
```